github-for-jira.sd.yml

```diff
@@ -63,4 +57,28 @@ environmentOverrides:
       remoteDebug: true
     config:
       environmentVariables:
+        APP_URL: https://github-for-jira.dev.services.atlassian.com
+        WEBHOOK_SECRET: development
```

Comment:
Not sure what this secret is used for, but if it's a secret, it shouldn't be in here, but instead stashed securely

Reply:
The APP_URL or the WEBHOOK_SECRET? The WEBHOOK_SECRET is publicly documented in CONTRIBUTING.md... so not very secret.

![Screen Shot 2021-06-07 at 11 21 27 am](https://user-images.githubusercontent.com/37155488/120947298-88d0e700-c782-11eb-825f-c4884510fa50.png)

Reply:
But the value of the WEBHOOK_SECRET variable should be secret, I'm sure. Anyways, it's good to list them in the service descriptor, but it should be commented out or something.